### DISCUSSION TITLE
[Esame 9 Settembre 2019] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#226] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/3d07e76d-36de-4ed0-9100-422d022ae226)

**Comment by LeoCosta02** [comment]
uguale al foglio 2 es 4
- #13
